BuildResidentialScheduleFile/resources/schedules.rb

```diff
@@ -900,26 +898,23 @@ def initialize_interior_lighting_schedule(args)
   # Generate occupancy schedules for sleeping, away, idle, EV presence and total occupancy.
   #
   # @param mkc_activity_schedules [Array<Matrix>] Array of matrices containing Markov chain activity states for each occupant
-  # @return [Hash] Hash containing arrays for sleep_schedule, away_schedule, idle_schedule, ev_occupant_presence, and present_occupants
+  # @return [Hash] Hash containing arrays for sleep_schedule, away_schedule, idle_schedule, ev_occupant_presence
   def generate_occupancy_schedules(mkc_activity_schedules)
     # States are: 0='sleeping', 1='shower', 2='laundry', 3='cooking', 4='dishwashing', 5='absent', 6='nothingAtHome'
     occupancy_arrays = {
       sleep_schedule: [],
       away_schedule: [],
       idle_schedule: [],
       ev_occupant_presence: [],
-      # Binary representation of the presence of occupant. Each bit represents presence of one occupant
-      present_occupants: []
     }
     @total_days_in_year.times do |day|
-      @steps_in_day.times do |step|
-        minute = day * 1440 + step * @minutes_per_step
+      1440.times do |minute_of_day|
+        minute = day * 1440 + minute_of_day
```

Comment:
I guess someone can figure it out from context, but would be better to define a class-level constant like `MinutesInDay=1440` and reference that everywhere.